### DISCUSSION TITLE
How save work on the IDE Remix

### DISCUSSION THREAD

**Discussion by wilonweb** [smartcontractkit/full-blockchain-solidity-course-py#441] [Q&A] [answered]
Hello, I follow Patrick Collins course on freeCodeCamp
But i don't know how to save my work one the online IDE https://remix.ethereum.org/.

**Comment by cromewar** [comment, marked as answer]
Hello @NinjaTurtles-cloud the project you are working on is saved automatically on for browser cache so the next time you open remix it'll remember what you were working on. If you want to save the project separately there is a button with the github icon to upload your project  there and if you want just a specific `.sol` just right click to it.

![image](https://user-images.githubusercontent.com/20199180/143446503-56d793a4-e712-421a-b093-fa25ed08bc0f.png)
